### PR TITLE
Feature: Work item files

### PR DESCRIPTION
**Pull request by osrjv**
This branch adds support for files in work items. No changes to existing keywords, but requires cloud support until it can be deployed.

### CODE REVIEW

File: packages/core/src/RPA/core/locators.py [hunk 1]
```diff
@@ -5,20 +5,15 @@
...
+    return os.path.join(dirname, filename)
+
 
-    if PROJECT_PATH_ENV in os.environ:
-        # locators.json is found at root of project, use environment var
-        # if available
-        return os.path.join(os.environ[PROJECT_PATH_ENV], DEFAULT_DATABASE_NAME)
-    else:
-        return DEFAULT_DATABASE_NAME
+DEFAULT_DATABASE = default_locators_path()
```
Comment (xylix) [inline packages/core/src/RPA/core/locators.py:16]: Good improvements in readability

File: docs/source/libraries/robocloud_items/index.rst [hunk 2]
```diff
@@ -36,11 +36,15 @@ overriden manually while developing an activity.
 Item structure
 ==============
 
-A work item's payload is JSON and allows storing anything that is
+A work item's data payload is JSON and allows storing anything that is
 serializable. This library creates an object with the key 'variables'
 that contains key-value pairs of a variable name and its contents.
 These variables can be exposed to the Robot Framework task to be used directly.
 
+Parallel to the data section, a work item can also contain files,
```
Comment (xylix) [inline docs/source/libraries/robocloud_items/index.rst:44]: Parallel might be an unclear term here? Maybe just a boring "In addition ..." could work?

File: docs/source/libraries/robocloud_items/index.rst [hunk 3]
```diff
@@ -78,13 +82,14 @@ the suite starts.
...
-        Log   Using variables from workspace ${workspace} for user ${user_id}
-
+    Save variables to Robocloud
+        Add work item file    orders.xlsx
+        Set work item variables    user=Dude    mail=address@company.com
+        Save work item
 
-In the following example the work item is modified locally and then saved
-back to Robocloud.
+Later in the process inside a different robot, we can use previously saved
```
Comment (xylix) [inline docs/source/libraries/robocloud_items/index.rst:90]: Do work items have any sort of namespacing / should you be careful to not collide names if you are running many robots in parallel?

File: docs/source/libraries/robocloud_items/index.rst (hunk 2 above)
Comment (osrjv) [reply docs/source/libraries/robocloud_items/index.rst:44]: A bit clunky, yes. Will fix.

File: docs/source/libraries/robocloud_items/index.rst (hunk 3 above)
Comment (osrjv) [reply docs/source/libraries/robocloud_items/index.rst:90]: Currently the work item data payload is just any JSON and the robot can freely modify it. It is however reserved for the execution and there will be no parallel access. Namespacing might or might not come when we start pushing reusable/shareable robots.

Comment (xylix) [reply docs/source/libraries/robocloud_items/index.rst:90]: Okay clear enough. Then the documentation from this PR should be sufficient

**Pull request merged**